wizeline/wize-q-front
# Feat/wip 253/about

#### What does this PR do?
- Add an `About` section
- Add a Pull Request template
- Replace the div-navbar with the Bootstrap Navbar implementation (which is responsive! 🎉)

#### How should this be manually tested?
As guest, nothing should've changed (only the navbar look)
1. Login
2. Check the `About` page right next to the WL logo
3. :tada:
Bonus: Use the developer tools of your browser to get the responsive version.

#### What are the relevant tickets?
Closes https://wizeline.atlassian.net/browse/WIP-253

#### Screenshots :camera:
**Main window**
![screen shot 2017-06-30 at 1 22 26 pm](https://user-images.githubusercontent.com/15201480/27749005-42ef89e8-5d97-11e7-9042-a6d764948554.png)

**About section**
![screen shot 2017-06-30 at 1 22 34 pm](https://user-images.githubusercontent.com/15201480/27749004-42edf182-5d97-11e7-96ff-48cdebd2bb50.png)

**BONUS: Responsive view**
![screen shot 2017-06-30 at 1 22 48 pm](https://user-images.githubusercontent.com/15201480/27749003-42ecc0c8-5d97-11e7-9c37-1803541d08f6.png)
